extra/hhvm.conf

```diff
@@ -19,7 +19,7 @@ hhvm.log.header = true
 hhvm.log.access[default][file] = /var/log/hhvm/access.log
 hhvm.log.access[default][format] = "%h %l %u %t \"%r\" %>s %b"
 hhvm.server.source_root = CTFPATH
-hhvm.repo.central.path = /var/cache/hhvm/hhvm.hhbc
+hhvm.repo.central.path = /var/run/hhvm/hhvm.hhbc
```

Comment:
`/var/run/hhvm/hhvm.hhbc` is created when the service starts and can be accessed by www-data without having to touch ownership or permissions. But `/var/cache/hhvm/hhvm.hhbc` will require to adjust permissions and the point is to persist after reboots, which the files under `/var/run` do not.